source/installation-guide/wazuh-server/step-by-step.rst

```diff
@@ -68,29 +68,24 @@ Installing the Wazuh manager
 
             # apt-get -y install wazuh-manager|WAZUH_MANAGER_DEB_PKG_INSTALL|
 
-#. Save the Wazuh indexer username and password into the Wazuh manager keystore using the wazuh-keystore tool: 
+Configuring the Wazuh indexer connection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Save the Wazuh indexer username and password into the Wazuh manager keystore using the wazuh-keystore tool:
```

Comment:
Is this step always required? I can see next step says to skip that particular step in case of not using vulnerability detection. Does it apply here as well? If both steps were to be skipped we could simply  explain it at the beginning of this subsection. But maybe it's necessary to always save credentials in the key store, even when not planning to use vulnerability detection.

Reply:
You are right, we can add a note.
It should be removed in the future when the rest of the modules use the indexer connector and filebeat is removed.